src/serve_data.js

```diff
@@ -259,8 +261,8 @@ export const serve_data = {
 
         let data = fetchTile.data;
         var param = {
-          long: bbox[0].toFixed(7),
-          lat: bbox[1].toFixed(7),
+          long: bbox[0],
+          lat: bbox[1],
```

Comment:
Is this change intentional. I think we purposely limited this to 7, but if it is causing an issue I am fine with reverting it.

Reply:
any update on this? If this is not needed and can be reverted I will merge moving the ungzip part of this.